cmds/contest/main.go

```diff
@@ -35,56 +36,72 @@ var (
 	flagServerID       = flag.String("serverID", "", "Set a static server ID, e.g. the host name or another unique identifier. If unset, will use the listener's default")
 	flagProcessTimeout = flag.Duration("processTimeout", api.DefaultEventTimeout, "API request processing timeout")
 	flagTargetLocker   = flag.String("targetLocker", inmemory.Name, "Target locker implementation to use")
+	flagInstanceTag    = flag.String("instanceTag", "", "Mandatory tag for this instance. Server will only operate on jobs with this tag and will add this tag to the jobs it creates.")
```

Comment:
Last nit: It says mandatory, what happens if this is not set?

Reply:
hm. actually, that's not what i meant. the flag is not mandatory, the tag on the jobs is mandatory if -instanceTag is specified. rephrased the description.

if the flag is not specified then instance will operate on all the jobs in the database with no additional filtering.